sklearn/neighbors/tests/test_lof.py

```diff
@@ -8,13 +8,14 @@
 from sklearn import neighbors
 import re
 import pytest
-from numpy.testing import assert_array_equal
 
 from sklearn import metrics
 from sklearn.metrics import roc_auc_score
 
 from sklearn.utils import check_random_state
 from sklearn.utils._testing import assert_array_almost_equal
```

Comment:
It remains a single occurence of `assert_array_almost_equal`. We can use `assert_allclose` for it and remove this import.